src/Roave/SecurityAdvisories/Version.php

```diff
@@ -195,8 +187,45 @@ private static function removeTrailingZeroes(int ...$versionNumbers) : array
         return [0];
     }
 
+    /**
+     * Bear in mind that we compare flags only when versions are equal
+     *
+     * @param Version $other
+     */
     private function compareFlags(self $other) : int
     {
+        $patchLiterals = [
```

Comment:
This is... Kinda terrible 😅

Still, good work on isolating it!

Reply:
threw it away.  
Hope your eyes are okay = )